docs/content/docs/grype/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Tech Preview - Grype Integration"
+linkTitle: "Access"
+weight: 1
+---
+
+Anchore Engine 0.10 includes a limited-functionality tech preview of an integration with [Grype](https://github.com/anchore/grype)
+for vulnerability scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Engine,
+but is provided in this release in a preview capacity so users can try it out.
+
+***Note:*** This tech preview is not intended for use in production environments. It should be installed in sandbox exnvironments,
+and is strictly provided here so users can get start to use it and get a sense for the functionality. It may not include all of the
+functionality of the legacy scanner. Please report any issues found with it on the
+[anchore-engine Github repo](https://github.com/anchore/anchore-engine/issues).
+
+### Installing
+As of 0.10.0, Anchore Engine can be configured to use either the legacy or grype vulnerability scanner. It is not possible to run
+both vulnerability scanners at the same time. This configuration is picked up at bootstrap, and cannot be changed ona running system. 
+
+#### Running with docker-compose
+1. Install or update to Anchore Engine 0.10.0.
+2. Add the following environment variable to the policy engine container section of the docker compose file:
+
+
+    policy-engine:
+      ...
+      environment:
+      ...
+      - ANCHORE_VULNERABILITIES_PROVIDER=grype
+
+3. Redeploy the services.
+
+#### Running with helm
+1. Install or update to Anchore Engine 0.10.0.
+2. Update the following value in `stable/anchore-engine/values.yaml`:
```

Comment:
Users won't use the git repo to get the chart. They'll create their own values.yaml. I think here we should link to the chart README on how to set this since the specifics are part of the chart